packages/orbit-components/src/Breadcrumbs/__tests__/index.test.js

```diff
@@ -1,77 +1,76 @@
 // @flow
+/* eslint-disable no-restricted-syntax */
 
 import * as React from "react";
-import { shallow } from "enzyme";
-import { render } from "react-dom";
-import { nullthrows } from "@adeira/js";
+import { render, screen, cleanup } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 
 import Breadcrumbs from "../index";
 import BreadcrumbsItem from "../BreadcrumbsItem";
 
-let screen;
-const body = nullthrows(document.body, "Expected document to have a body");
-
-beforeEach(() => {
-  screen = document.createElement("div");
-  body.appendChild(screen);
-});
-
-afterEach(() => {
-  body.removeChild(screen);
-});
+// jsdom doesn't support @media queries, so for <Hide> the computed value of
+// "display" is "none", which @testing-library/react (impressively) detects as
+// inaccessible, so we're replacing <Hide> with a dummy component
+jest.mock("../../Hide", () => ({ children }) => children);
```

Comment:
This was fascinating to me 😮 none of my `ByRole` queries were working, then I figured out why.